src/main/java/com/sinthoras/visualprospecting/mixinplugin/TargetedMod.java

```diff
@@ -11,6 +11,7 @@
     // Replace with your injected mods here, but always keep VANILLA:
     VANILLA("Minecraft", "unused", true),
     GREGTECH("GregTech", "gregtech", true),
+    IFU("I will find you", "ifu", true),
```

Comment:
`"ifu"` is the begin of a jar name. ANY jar name.  On Linux this will be case sensitive (no windows). The original build script produces a `IFU-1.8.jar`. It should match both. Maybe adapt your build script PR?

Reply:
Yeah, I'll do that, hope nobody else relies on it being upper case.

Is there a way to make this case insensitive?

Reply:
Check loadJar in MixinPlugin. Maybe it is easy to do.

Reply:
made a method which should be case insensitive and only take prefixes into account